### PR TITLE
Feature/update home page

### PR DESCRIPTION
**Pull request by ADavidson02**
## PR Template 

### Is this a fix or a feature?  
* Feature 

### What is the change? 
* Complete functionality of fetching data from firebase 

### Where should the reviewer start? 
* src/Archive/Archive.js line 3
* src/Card/Card.js line 8
* src/Form/Form.js line 4 

### How should this be tested?
* Load the /archive page and see if advice is loaded

### CODE REVIEW

File: src/Form/Form.js [hunk 1]
```diff
@@ -19,44 +22,52 @@ class Form extends Component {
...
+            type="text"
+            name="email"
+            placeholder="Email"
+            value={this.state.email}
+            onChange={(e) => this.handleChange(e)}
           />
         </div>
         <div>
-          <button onClick={ this.submitAdvice }>Save Advice</button>
+          <Link to="/archive">
```
Comment (atownse) [inline src/Form/Form.js:64]: Good, we have the route to archive

File: src/Archive/Archive.js [hunk 2]
```diff
@@ -1,26 +1,55 @@
...
-  render(){
-    return(
+  componentDidMount() {
+    console.log("this", this);
+    getAdviceSlips()
+      .then((response) => {
+        this.setState({ adviceSlips: response.data });
+      })
+      .catch((error) => {
+        console.log(error);
```
Comment (atownse) [inline src/Archive/Archive.js:22]: We will need to fix this to update state for an error message to display

Comment (atownse) [inline src/Archive/Archive.js:22]: We will need to fix this to update state for an error message to display

**Review by atownse — APPROVED** [review]

**Review by Estaffieri — APPROVED** [review]
Thanks for updating this to our slimmed down specs for the project.

**Pull request merged**